pkg/master/master.go

```diff
@@ -586,12 +593,12 @@ func (m *Master) initV1ResourcesStorage(c *Config) {
 }
 
 // NewBootstrapController returns a controller for watching the core capabilities of the master.
```

Comment:
And/or would you prefer that `NewBootstrapController` require

1. the passed in argument is non-nil
1. Reconciler is non-nil
1. Interval is > 0

and that we do defaulting upwards in the stack?